gradlew

```diff
@@ -64,10 +64,6 @@ case "`uname`" in
     ;;
 esac
 
-if [ ! -e $APP_HOME/gradle/wrapper/gradle-wrapper.jar ]; then
-    curl -o $APP_HOME/gradle/wrapper/gradle-wrapper.jar https://raw.githubusercontent.com/gradle/gradle/v5.4.1/gradle/wrapper/gradle-wrapper.jar
-fi
-
```

Comment:
Is this no longer needed at all? Did the usage of the gradle-wrapper.jar for gradle 5.6.4? I’m surprised that this line is no longer needed given that this jar is still referenced on the old line 71 / new line 67 (https://github.com/apache/iceberg/pull/2816/files#diff-e9721dc750619a21053ddea8a5d04929a608877d8c5daec1b57d243d3424e745R67)

Have you tested that this works on fresh system without the gradle wrapper jar present?

Reply:
The standard setup for a Gradle build is to include the Gradle wrapper jar in the project. I just noticed that this is not the case for Apache Iceberg. Can we check in the wrapper to make it a more standard setup?

Reply:
I think I need to append this PR with the Gradle wrapper jar in another commit. Will do it now.